src/covid19model/visualization/optimization.py

```diff
@@ -0,0 +1,146 @@
+import datetime
+import random
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+import pandas as pd
+import numpy as np
+
+
+# From Color Universal Design (CUD): https://jfly.uni-koeln.de/color/
+colorscale_okabe_ito = {"orange" : "#E69F00", "light_blue" : "#56B4E9",
+                        "green" : "#009E73", "yellow" : "#F0E442",
+                        "blue" : "#0072B2", "red" : "#D55E00",
+                        "pink" : "#CC79A7", "black" : "#000000"}
+
+# covid 19 specific parameters
+plt.rcParams.update({
+    "axes.prop_cycle": plt.cycler('color',
+                                  list(colorscale_okabe_ito.values())),
+    "font.size": 15,
+    "lines.linewidth" : 3,
+    "axes.spines.top": False,
+    "axes.spines.right": False,
+    "ytick.major.left": True,
+    "axes.grid": True
+})
```

Comment:
@jorisvandenbossche how to optimally manage these in a package so they can be applied to each of the files/plots?

Reply:
I would put this in the `__init__.py` file, and then once you import anything from the visualization functions, then this will always be run on import ?

Reply:
ok, that's what I would think as well.